sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorAdministrationClient.cs

```diff
@@ -142,28 +142,33 @@ public virtual Response<DataFeed> GetDataFeed(string dataFeedId, CancellationTok
         /// <returns>An <see cref="AsyncPageable{T}"/> containing the collection of <see cref="DataFeed"/>s.</returns>
         public virtual AsyncPageable<DataFeed> GetDataFeedsAsync(GetDataFeedsOptions options = default, CancellationToken cancellationToken = default)
         {
-            string name = options?.Filter?.Name;
-            DataFeedSourceKind? sourceKind = options?.Filter?.SourceKind;
-            DataFeedGranularityType? granularityType = options?.Filter?.GranularityType;
-            DataFeedStatus? status = options?.Filter?.Status;
-            string sourceKindValue = sourceKind.HasValue ? sourceKind.Value.ToString() : null;
-            string granularityTypeValue = granularityType.HasValue ? granularityType.Value.ToString() : null;
-            string statusValue = status.HasValue ? status.Value.ToString() : null;
-            RequestContext context = new RequestContext()
-            {
-                CancellationToken = cancellationToken,
-            };
-            string creator = options?.Filter?.Creator;
-            int? skip = options?.Skip;
-            int? maxPageSize = options?.MaxPageSize;
+            using DiagnosticScope scope = ClientDiagnostics.CreateScope($"{nameof(MetricsAdvisorAdministrationClient)}.{nameof(GetDataFeeds)}");
```

Comment:
I believe we don't want to create this scope here.  Are any network calls sent before it closes?

Reply:
I think this scope is needed because we do serialization here

Reply:
No, there is no network call sent in this scope. Below is the workflow of this scope:

Create Scope -> Serialize Input Model -> Create Pageable -> Close Scope

I'll update paging methods to remove outer scope.